src/utils/calls/farms.ts

```diff
@@ -5,40 +5,40 @@ const options = {
   gasLimit: DEFAULT_GAS_LIMIT,
 }
 
-export const stakeFarm = async (masterChefContract, pid, amount) => {
+export const stakeFarm = async (masterChefContract, pid, amount, gasPrice) => {
```

Comment:
Rather than refactor these functions into hooks, I opted to just pass `gasPrice` into the function. I expect these will be refactored in the coming weeks anyway.